rest-api/src/main/java/gov/cms/qpp/conversion/api/model/HealthCheck.java

```diff
@@ -0,0 +1,62 @@
+package gov.cms.qpp.conversion.api.model;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Response sent when a health check request is received
+ */
+public class HealthCheck {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)